gnmipath/internal/metadata/generated_status.go

```diff
@@ -0,0 +1,25 @@
+// Code generated by mdatagen. DO NOT EDIT.
```

Comment:
Can we add a `README.md` file that lists out which files are autogenerated and anything that we should know about regenerating them?